tests/test_utils.py

```diff
@@ -486,3 +492,167 @@ def template(x):
             template(3)
 
         assert str(recorder) == expected_output
+
+
+@qml.template
+def dummy_template(wires):
+    """Dummy template for inv tests."""
+    for wire in wires:
+        qml.RX(1, wires=[wire])
+        qml.RY(-1, wires=[wire])
```

Comment:
It might be worth adding several operations of the same type as well, perhaps with alternating order as well

Reply:
Will not be an issue. The operations are always uniquely identified via their ID:
```python
>>> import pennylane as qml
>>> qml.PauliX(0) == qml.PauliX(0)
False
```

Reply:
Oh you're right! Back when I made this comment I was assuming that the example with the queue outside of context was not working as it had at least 3 of the same operations. Now it doesn't seem to be a problem, right?

Reply:
Exactly, in that example I was wrongly assuming that when inside a context all operations that are given for inversion have also been queued in this context